### PR TITLE
Add slugify recipe

### PR DESCRIPTION
**Pull request by proinsias**
Due to a Windows/Python 3 [issue](aerkalov/ebooklib#104) with the ebooklib dependency, skip the Windows/Python 3 builds for now.

### CODE REVIEW

File: recipes/slugify/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - slugify
+
+about:
+  home: http://github.com/zacharyvoase/slugify
+  license: Public Domain
+  license_family: Other
+  # license_file: UNLICENSE # When added to MANIFEST.in – https://github.com/zacharyvoase/slugify/pull/4
```
Comment (jakirkham) [inline recipes/slugify/meta.yaml]: xref: https://github.com/zacharyvoase/slugify/pull/4

Comment (jakirkham) [inline recipes/slugify/meta.yaml]: Technically we don't have to, but I agree that we should. Thanks for putting that PR together. 👍

File: recipes/slugify/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{% set name = "slugify" %}
+{% set version = "0.0.1" %}
+{% set md5 = "557a7e401ea3c9716d29389c56cf82c5" %}
```
Comment (jakirkham) [inline recipes/slugify/meta.yaml]: Could use `sha256` from [pypi.io]( https://pypi.io/ ).

File: recipes/slugify/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [win and py3k]
```
Comment (jakirkham) [inline recipes/slugify/meta.yaml]: Why do we need to skip these?

Comment (proinsias) [inline recipes/slugify/meta.yaml]: Due to a Windows/Python 3 issue with the ebooklib dependency, we skip the Windows/Python 3 builds for now.

File: recipes/slugify/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - slugify
+
+about:
+  home: http://github.com/zacharyvoase/slugify
+  license: Public Domain
+  license_family: Other
+  # license_file: UNLICENSE # When added to MANIFEST.in – zacharyvoase/slugify#4
```
Comment (jakirkham) [inline recipes/slugify/meta.yaml]: xref: https://github.com/zacharyvoase/slugify/pull/4

As mention elsewhere, not technically require in this case. Then again, not a bad idea either.

**Comment by jakirkham** [comment]
Thanks @proinsias.

File: recipes/slugify/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/slugify/meta.yaml]: Not seeing `ebooklib` as a dependency. Maybe we should enable Windows builds completely. Please see PR ( https://github.com/conda-forge/slugify-feedstock/pull/1 ).

**Pull request merged**